edit/resources.cattle.io.backup.vue

```diff
@@ -91,7 +92,7 @@ export default {
     },
 
     chartNamespace() {
-      const BRORelease = this.apps.filter(release => get(release, 'spec.name') === 'rancher-backup')[0];
```

Comment:
@mantis-toboggan-md This should (and same in restore) come from config/product/backup NAME